src/Search.php

```diff
@@ -4744,8 +4744,52 @@ public static function addWhere($link, $nott, $itemtype, $ID, $searchtype, $val,
 
         switch ($searchtype) {
             case "notcontains":
-                $nott = !$nott;
```

Comment:
reverted when we reach
https://github.com/glpi-project/glpi/blob/10.0/bugfixes/src/Search.php#L5405